index.js

```diff
@@ -1,12 +1,11 @@
 module.exports = {
-  plugins: ['html', 'json'],
   extends: [
-    '@fs/eslint-config-frontier-react/es6',
-    '@fs/eslint-config-frontier-react/react',
-    '@fs/eslint-config-frontier-react/jest',
-    '@fs/eslint-config-frontier-react/test-helpers',
-    // '@fs/eslint-config-frontier-react/noFixRules',
-    '@fs/eslint-config-frontier-react/dont-need-lodash',
-    '@fs/eslint-config-frontier-react/prettierSetup',
+    './react',
+    './es6',
+    './typescript',
```

Comment:
Again, I can't pin point the exact issue, but I'm pretty sure that when we defaulted to using typescript all the time, that some annoying/unwanted behavior was occurring. It is possible that we didn't have things setup for typescript to only run .ts/.tsx files, but I'd want to make sure this was testing/verified in a non typescript react app.

Reply:
yep I tested in all  tw-blue apps which includes 2 typescript apps and 3 js apps and had no ts specific issues for the js apps.